e2e/loadgen/cmd/cmd.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"github.com/omni-network/omni/e2e/loadgen/app"
+	libcmd "github.com/omni-network/omni/lib/cmd"
+	"github.com/omni-network/omni/lib/log"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// New returns a new root cobra command that runs the xcall load generator.
+func New() *cobra.Command {
```

Comment:
I'd actually just follow the `TestJoinNetwork` example. Not sure a whole app is required. Lets keep it simple.